### DISCUSSION TITLE
I2P Live CD

### DISCUSSION THREAD

**Discussion by l-n-s** [PurpleI2P/PurpleI2P#13] [Ideas]
General topic to discuss I2P Live CD development

**Comment by l-n-s** [comment]
I'm planning to create live CD with i2pd and some apps, not very paranoid setup ATM. More like Tails, then Whonix.

Plan to use Debian Stable for it. More info later.

**Comment by orignal** [comment]
They are not interested

**Comment by majestrate** [comment]
Tails is not interested, whonix and sgos are.

**Comment by orignal** [comment]
The said explicitly they don't want i2pd there.

**Comment by majestrate** [comment]
you can try to convince them (tails) to add i2pd but I doubt they will.
